app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -26,6 +26,8 @@ def button
     # Handle Toolbar Policy Tag Button
     @refresh_div = "main_div" # Default div for button.rjs to refresh
     tag(self.class.model) if params[:pressed] == "#{params[:controller]}_tag"
+    assign_policies(ContainerImage) if params[:pressed] == "container_image_protect"
+    check_compliance_images if params[:pressed] == "container_image_check_compliance"
```

Comment:
This is repetitive.  I assume we are pressing a button, but we're checking several things based on that.  We should be instantiating a button and calling pressed or something on it. The class of button can call these methods to do the work for that button press.

Not for this PR but clearly this gets really complicated if we need to do this with lots of button types.